Storage/StorageTests/CoreData/MigrationTests.swift

```diff
@@ -3228,4 +3315,34 @@ private extension MigrationTests {
 
         return campaign
     }
+
+    /// Inserts a `BlazeTargetLanguage` entity, providing default values for the required properties.
+    @discardableResult
+    func insertBlazeTargetLanguage(to context: NSManagedObjectContext) -> NSManagedObject {
+        let language = context.insert(entityName: "BlazeTargetLanguage", properties: [
+            "id": "en",
+            "name": "English"
+        ])
+        return language
+    }
+
+    /// Inserts a `BlazeTargetDevice` entity, providing default values for the required properties.
+    @discardableResult
+    func insertBlazeTargetDevice(to context: NSManagedObjectContext) -> NSManagedObject {
+        let device = context.insert(entityName: "BlazeTargetDevice", properties: [
+            "id": "mobile",
+            "name": "Mobile"
+        ])
+        return device
+    }
+
+    /// Inserts a `BlazeTargetTopic` entity, providing default values for the required properties.
+    @discardableResult
+    func insertBlazeTargetTopic(to context: NSManagedObjectContext) -> NSManagedObject {
+        let device = context.insert(entityName: "BlazeTargetTopic", properties: [
+            "id": "IAB1",
+            "name": "Arts & Entertainment"
+        ])
+        return device
```

Comment:
nit: This can be renamed as `topic`
```suggestion
        let topic = context.insert(entityName: "BlazeTargetTopic", properties: [
            "id": "IAB1",
            "name": "Arts & Entertainment"
        ])
        return topic
```